code/modules/keybindings/bindings_atom.dm

```diff
@@ -22,3 +22,5 @@
 		keybind_face_direction(movement_dir)
 	else
 		user?.Move(get_step(src, movement_dir), movement_dir)
+
+	return !!movement_dir //true if there was actually any player input
```

Comment:
still think this is measuring the wrong thing, specially as a moving average

Reply:
i suppose it would be more accurate to only return this if a Move() occured.